mnefun/_mnefun.py

```diff
@@ -1772,3 +1777,28 @@ def gen_html_report(p, subjects, structurals, raw=True, evoked=True,
                             pattern=patterns)
         report_fname = get_report_fnames(p, subj)[0]
         report.save(report_fname, open_browser=False, overwrite=True)
+
+
+def plot_raw_psd(p, subjects, tmin=0, fmin=2, n_fft=2048, n_jobs=1):
+    """Plot data power for all available raw data files for a subject"""
+    for subj in subjects:
+        for file_type in ['raw', 'sss']:
+            fname = get_raw_fnames(p, subj, which=file_type, erm=False)
+            if len(fname) < 1:
+                raise Warning('Unable to find %s data file.' % file_type)
+            raw = io.Raw(fname, preload=True, allow_maxshield=True)
```

Comment:
put this in a `with warnings.catch_warnings(record=True):` context manager so that the maxfilter warning is squashed